frappe/utils/oauth.py

```diff
@@ -232,7 +232,7 @@ def update_oauth_user(user, data, provider):
 		user.update({
 			"doctype":"User",
 			"first_name": get_first_name(data) or get_email(data),
-			"last_name": get_last_name(data),
+			"last_name": get_last_name(data) or "",
```

Comment:
```suggestion
			"last_name": get_last_name(data),
```
Return empty string from the get_last_name function

Reply:
@hrwX Not required in the current aws login flow. So we can close this PR.